bindings/c/test/fdb_api.hpp

```diff
@@ -722,6 +722,20 @@ inline void selectApiVersion(int version) {
 	}
 }
 
+inline Error selectApiVersionCappedNothrow(int version) {
+	if (version < 720) {
```

Comment:
This seems to be another place where we now seem to need to update the version with each release... We should probably generate these from CMake

Reply:
Not in this case. As I understand this is a change of the tenant prefix specifically from 7.1 to 7.2.